medical_patient_disease/views/medical_patient_disease_view.xml

```diff
@@ -162,7 +162,7 @@
                             </group>
                         </group>
                     </group>
-                    <group name="state" attrs="{'invisible': [('id','=',False)]}">
```

Comment:
Someone on Stack Overflow was saying that you couldn't have a different view for create & edit. They were suggesting workarounds like checking the value of created_at to see if the user was creating or editing. But would love to hear other solutions!